src/test/java/ch/jalu/collectionbehavior/model/ListMethod.java

```diff
@@ -0,0 +1,54 @@
+package ch.jalu.collectionbehavior.model;
+
+import java.util.List;
+
+/**
+ * Enum representing various methods available on a {@link List}.
+ */
+public enum ListMethod {
+
+    /** {@link List#add(Object)} */
+    ADD,
```

Comment:
ADD is used for `add(int, E)` also -> separate enum entry (or minimally, adapt javadoc)